coa-helm/templates/cbas-ui.yaml

```diff
@@ -28,7 +28,12 @@ data:
       "leoUrlRoot" : "{{ .Values.leonardo.url }}",
       "cbasUrlRoot" : "{{ .Values.relay.path }}/cbas",
       "cromwellUrlRoot" : "{{ .Values.relay.path }}/cromwell",
-      "wdsUrlRoot" : "{{ .Values.relay.path }}/wds",
-      "wdsInstanceId" : "{{ .Values.persistence.workspaceManager.workspaceId }}"
+      "wdsAppTypeNames": ["WDS", "CROMWELL"],
+      "wdsInstanceId" : "{{ .Values.persistence.workspaceManager.workspaceId }}", #Same as workspace ID
+      "storageAccount" : "{{ .Values.persistence.storageAccount }}",
+      "blobContainer" : "{{ .Values.persistence.blobContainer }}",
+      "workspaceManagerRoot" : "{{ .Values.persistence.workspaceManager.url }}",
+      "workspaceId" : "{{ .Values.persistence.workspaceManager.workspaceId }}",
+      "containerResourceId" : "{{ .Values.persistence.workspaceManager.containerResourceId }}"
```

Comment:
It looks like some of these items didn't end up getting used in CBAS, should they be removed here?

Reply:
Totally, removed the unneeded ones. In order to get a sas token from WSM, I believe we need the `workspaceManagerRoot`, `workspaceId`, and `containerResourceId`. Removed `blobContainer` and `storageAccount` as they are not necessary.